packages/agoric-cli/src/commands/perf.js

```diff
@@ -86,12 +86,12 @@ export const makePerfCommand = async logger => {
       void watchForSatisfied();
 
       // now execute
-      let cmd = `wallet-action --allow-spend "$(cat ${opts.executeOffer})"`;
+      let cmd = ['wallet-action', '--allow-spend', payloadStr];
       if (opts.keyringBackend) {
-        cmd = cmd.concat(' --keyring-backend ', opts.keyringBackend);
+        cmd = cmd.concat([`--keyring-backend=${opts.keyringBackend}`]);
```

Comment:
non-blocking request: make `const cmd` and  append each item:
```suggestion
        cmd.push(`--keyring-backend=${opts.keyringBackend}`);
```
now that it's an array, the symbol doesn't need to be rebound to amend it. it also makes it read more clearly.